docs/releases/v1.5.0.md

```diff
@@ -0,0 +1,55 @@
+# Disaster Recovery Core Module version 1.5.0
+
+SIGHUP team maintains this module updated and tested. That is the main reason why we worked on this new release.
+With the Kubernetes 1.19 release, it became the perfect time to start testing this module against this Kubernetes
+release. VMWare continues doing amazing work with Velero. This has become a standard to backup and restore
+Kubernetes' clusters. With the release of its latest release, we want to include the new release in this core module.
+Continue reading the [Changelog](#changelog) to discover them:
+
+## Changelog
+
+- Update Velero. From version `1.3.1` to `1.5.1`.
+  - Add Velero Grafana dashboard.
+  - Increase Velero memory limit to 1Gi: [https://github.com/vmware-tanzu/velero/issues/2840](https://github.com/vmware-tanzu/velero/issues/2840)
+- Update Velero-restic. From version `1.3.1` to `1.5.1`.
+- Update Velero plugins *(AWS, GCP, and Azure)*. From version `1.0.1` to `1.1.0`.
+- Update Velero on-premise Minio server. From version `RELEASE.2020-01-25T02-50-51Z` to `RELEASE.2020-09-17T04-49-20Z`.
+- Change cloud credentials terraform output to be a Kubernetes manifests instead of a plain text file.
+  - Applies to the three different providers *(AWS, GCP, and Azure)*.
+
+## Upgrade path
+
+## Katalog
+
+To upgrade this core module from `v1.4.0` to `v1.5.0`, you need to download this new version, then apply the
+`kustomize` project. No further action is required.
+
+```bash
+$ kustomize build katalog/velero-on-prem | kubectl apply -f -
+# or
+$ kustomize build katalog/velero-aws | kubectl apply -f -
+# or
+$ kustomize build katalog/velero-gcp | kubectl apply -f -
+# or
+$ kustomize build katalog/velero-azure | kubectl apply -f -
+# and/or
+$ kustomize build katalog/velero-restic | kubectl apply -f -
+```
+
+### Terraform modules
+
+Before this `v1.5.0` release, you runned commands like:
+
+```bash
+$ terraform output cloud_credentials > /shared/cloud_credentials.config
+$ kubectl create secret generic cloud-credentials --from-file=cloud=/shared/cloud_credentials.config --dry-run -o yaml | kubectl apply -f - -n kube-system
```

Comment:
Actually, we were using kustomize `secretGenerator` and this could be a breaking change for our existing customers relaying on that behavior.

Reply:
You only need to avoid creating the secret, then apply the resource

Reply:
The output from the terraform module is a Kubernetes secret yaml ready to be applyed

Reply:
we could output both to keep backwards compatibility, keep the old output with the same name and add the new one with another name like `cloud_credentials-yaml` or something like that..